### PR TITLE
Remove redundant let error

### PR DESCRIPTION
**Pull request by mk13**
Code will be broken temporarily until this is merged.

### CODE REVIEW

File: analyzer_plugin/lib/ast.dart [hunk 1]
```diff
@@ -195,6 +195,8 @@ class TextAttribute extends AttributeInfo {
 }
 
 class Mustache extends AngularAstNode {
+  static const EMPTY_LIST = const <Mustache>[];
```
Comment (liuming0) [inline analyzer_plugin/lib/ast.dart]: use camelCase.

Comment (mk13) [inline analyzer_plugin/lib/ast.dart]: There's a bit of SCREAMING_CAPS already happening within the code, so I'll just leave this to stay consistent. (https://www.dartlang.org/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names)

Comment (scheglov) [inline analyzer_plugin/lib/ast.dart]: FWIW, in Analyzer we at the end decided to use explicit `const <TypeName>[]` - it is not longer and not more expensive. We didn't migrate, but try to write new code in this style.

Comment (mk13) [inline analyzer_plugin/lib/ast.dart]: Oh sure, good to know. I'll go ahead and add those changes into this PR as well then.

**Pull request merged**